### DISCUSSION TITLE
Which continent has the least 🏜️🐫?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#867] [Galaxy Trivia] [answered]
**Bonus:** What has the most?

**Comment by PatilHarshh** [comment, marked as answer]
Africa. (Most)
Antarctica (Least)

@ParanoidUser

**Reply by ParanoidUser** [reply]
Congratulations! 👏👏 You have just earned the prestigious [Galaxy Brain](https://github.com/PatilHarshh?tab=achievements&achievement=galaxy-brain) badge. Keep up the great work and unlock the Bronze 🥉 badge next by answering 8 questions. Don't forget to post your own questions too! 😊🎉

**Reply by PatilHarshh** [reply]
Yay !! papu Pass ho gaya!!!  
thankyou @ParanoidUser ;)

**Comment by peterbaccaro** [comment]
The continent with the least population of desert camels is **Antarctica**. This is because Antarctica is a frozen continent with extreme cold temperatures and very limited vegetation, making it an unsuitable habitat for camels. Camels are primarily found in the deserts of Africa, the Middle East, and Asia.

The continent with the most population of desert camels is **Africa**, particularly the dromedary camel (one-humped camel). Camels are found in several African countries, including Sudan, Somalia, Mali, Niger, Chad, and Egypt, among others.
